docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,174 @@
+</div>
+```
+
+## Text accessibility
+Text accessibility involves making content readable and understandable. You can achieve text accessibility by using the following:
+* Color contrast
+* Text headings
+* Text size and alignment
+* Text links
+
+### Color contrast
+Color contrast is an essential factor in visual accessibility. Use contrasting colors for better readability, along with keeping headings clear and concise.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> According to [WCAG 2.1 (Web Content Accessibility Guidelines)](https://www.w3.org/TR/WCAG21/), your eCommerce website must have a minimum color contrast ratio of at least 4.5.1 for standard text and 3:1 for large text. We recommend presetting these guidelines, using [Lighthouse](https://developers.google.com/web/tools/lighthouse) or Chrome's [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb/reviews?hl=en) to ensure you meet WCAG 2.1 guidelines. 
+
+</div>
+</div>
+</div>
+
+Pure black text on a white background can cause eye strain. Instead, try black text on light blue, pale green, or gray background. The following example demonstrates good color contrasting and has a 14.9.1 contrast ratio.
+
+**Good contrast ratio**
+![Good contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_good_color_contrast.png "Good contrast")
+
+It is essential to avoid colors that are too close together. For example, avoid green and black, green and gray, and green and red. Alternatively, it is good practice to use light text on dark backgrounds or dark text on light backgrounds. The example below shows poor color contrasting because the black text is on a green background. The color constrast ratio is 3.8.1. It would have been better to use white text on a dark green background.
+
+**Poor contrast ratio**
+![Poor contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_poor_color_contrast.png "Poor contrast")
+
+### Text headings
+Text headings organize and provide structure for a page, helping users to understand its contents. Headings should be logical, clear, and concise. The following example demonstrates the incorrect and correct structure of text headings.
+
+![Text headings](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_text_headings.png "Text headings")
+
+Your main heading should be H1 and indicate the main content. The incorrect example displays the main heading as H2, which you should only use for subtopics. In addition, don't skip or alter headings as it could cause a fragmented experience, as shown above. H3 is out of order and should follow H2. The correct structure displayed above on the right shows heading levels from h1 to h3 communicating more specific information about the previous section.
```

Comment:
Be sure your capitalization stays consistent from H1 to h3.
For the sake of clarity and brevity, I recommend only discussing what developers should do when it comes to headings instead of discussing what they shouldn't do at length.

Example:
When structuring a page, work your way from largest to smallest heading. Avoid breaking the sequence of headings to prevent a fragmented accessibility experience.